### PR TITLE
Add GitHub authorization, add flash messages

### PR DESCRIPTION
**Pull request by olegsobchuk**
story https://www.pivotaltracker.com/story/show/110311618

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+  def create
+    account = Account.send(env['omniauth.auth']['provider'], env['omniauth.auth'])
+    if account
+      session['account_id'] = account.id
+      flash[:success] = 'Logged in!'
+    else
+      flash[:error] = 'Ferification was faild!'
```
Comment (Alexey-Lukin) [inline app/controllers/sessions_controller.rb]: Verification was failed!

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,16 @@
 module ApplicationHelper
+  def alert(flash_type)
+    case flash_type.to_sym
+    when :success
+      'alert-success'
+    when :error
+      'alert-error'
+    when :prymary
+      'alert-prymary'
```
Comment (Alexey-Lukin) [inline app/helpers/application_helper.rb]:
```
when :primary
  'alert-primary'
```

File: app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -0,0 +1,12 @@
+doctype html
+html
+  head
+    title Benchmarkit
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+
+  body
+  = render 'layouts/session'
```
Comment (Alexey-Lukin) [inline app/views/layouts/application.html.slim:10]: Is it correct indentation here?

File: config/routes.rb [hunk 4]
```diff
@@ -1,7 +1,15 @@
 Rails.application.routes.draw do
   root 'battles#new'
 
+  resources :accounts, only: [] do
+    resources :battles, only: [:show, :create, :restroy]
```
Comment (Alexey-Lukin) [inline config/routes.rb]: :destroy

File: config/secrets.yml [hunk 5]
```diff
@@ -1,19 +1,14 @@
-# Be sure to restart your server when you modify this file.
+defaults: &defaults
```
Comment (Alexey-Lukin) [inline config/secrets.yml]: we don't need defaults section

File: app/views/layouts/application.html.slim (hunk 3 above)
Comment (olegsobchuk) [inline app/views/layouts/application.html.slim:10]: it will be changed, when we add header

**Pull request merged**